site/docs/foundations/duration.mdx

```diff
@@ -20,21 +18,78 @@ There are four durations available to choose from: Instant, Perceptible, Notable
 | `--salt-duration-notable`     | 1000                    |
 | `--salt-duration-cutoff`      | 10000                   |
 
-### Instant
+## Instant (0-100ms)
 
-The immediate response by the system to a users interaction, such as hovering over a button, clicking on a radio button, typing into an editable field, or dragging an object.
+An instant duration is the immediate response by the system to a user’s interaction, within a time frame of 0-100ms. The response should take no more than 100ms to complete. The types of interactions that can trigger an instant response include hovering over a button, clicking on a radio button, typing into an editable field, or dragging an object. No feedback is required as the system response itself is the feedback.
 
-### Perceptible
+<Image
+  src="/img/foundations/duration-instant.svg"
+  alt="Example of duration instant"
+/>
 
-A short but detectable delay between a user's action and the system's response. The user should be aware of the gap but retain the feeling of control. In essence, the delay should feel acceptable. Tooltips and overlays are examples of perceptible delays.
+## Perceptible (300ms)
 
-### Notable
+A perceptible duration is a short, but detectable delay between a user's action and the system's response. The user should be aware of the gap but retain the feeling of control. This implies a shared control between the user and the interface itself—the user expects the system to process the interaction they have enabled the system to do. However, the delay should feel acceptable, and therefore maintain the user’s attention and productivity. This aligns with the “[Doherty Threshold](https://lawsofux.com/doherty-threshold/)” principle which dictates that a user experience turns from painful to addictive after the system feedback time drops below 400ms. Tooltips and overlays are examples of perceptible delays.
 
-With a notable delay, the user is aware the system is slow to respond. A feeling of loss of control may occur, which is why it's important the delay is clearly communicated. Notable delays may be required in order to load a data grid or retrieve search results.
+<Image
+  src="/img/foundations/duration-perceptible.svg"
+  alt="Example of duration perceptible"
+/>
 
-### Cut-off
+Perceptible is recommended for when you want to reveal secondary guidance or information to your user, but it doesn't warrant a permanent place within the interface. Content typically has a relatively high level of urgency, so that users can quickly find solutions to their problems and avoid getting frustrated with the system.
```

Comment:
```suggestion
Perceptible duration is recommended for when you want to reveal secondary guidance or information to your user, but it doesn't warrant a permanent place within the interface. Content typically has a relatively high level of urgency, so that users can quickly find solutions to their problems and avoid getting frustrated with the system.
```